gnucash/import-export/aqb/gnc-gwen-gui.c

```diff
@@ -1421,8 +1482,47 @@ getpassword_cb(GWEN_GUI *gwen_gui, guint32 flags, const gchar *token,
     gchar *password = NULL;
     gboolean is_tan = (flags & GWEN_GUI_INPUT_FLAGS_TAN) != 0;
 
+    #ifdef GWENHYWFAR5
+    int opticalMethodId;
+    const char *mimeType = NULL;
+    const char *pChallenge = NULL;
+    uint32_t lChallenge = 0;
+    #endif
+
     g_return_val_if_fail(gui, -1);
 
+    #ifdef GWENHYWFAR5
+    // cf. https://www.aquamaniac.de/rdm/projects/aqbanking/wiki/ImplementTanMethods
+    if(is_tan && methodId == GWEN_Gui_PasswordMethod_OpticalHHD)
+    {
+      /**
+      * TODO: How to handle Flicker code (use WebView and JS???)
```

Comment:
GdkPixbufAnimation would be more robust, particularly on Microsoft Windows.

Reply:
I thought of the WebView because there seems to exist a JS implementation of the "FlickerTAN"

Reply:
There does, but the code for setting up a webview is currently limited to the reports section and I think it should stay that way. Plus we have [ongoing](https://bugs.gnucash.org/show_bug.cgi?id=797283) [issues](https://bugs.gnucash.org/show_bug.cgi?id=797396) with WebKit on Microsoft Windows. The AQBanking blog claims it's not that hard to implement.

Reply:
Okay, that is a reason. I had a look into the "reference" app Aqfinance and found an implementation in cpp and fox. That should do for a template...

Reply:
OK. I suggest you do that as a separate PR.

Reply:
Ok